File: Tareas_FdP/Tarea02vscode/Ejercicio01.java
import java.io.BufferedReader;
import java.io.InputStreamReader;
import java.io.IOException;

public class Ejercicio01 {
    public static void main(String[] args) throws IOException {
        BufferedReader reader = new BufferedReader(new InputStreamReader(System.in));

        double interes = 0.02;
        
        System.out.println("Ingrese el monto a invertir: ");
        double capital = Double.parseDouble(reader.readLine());

        System.out.println("Ingrese el número total de días del mes a considerar: ");
        int dias = Integer.parseInt(reader.readLine());

        double ganancia = capital * dias * interes;

        System.out.println("La ganancia por cobrar después del mes es de: " + ganancia);
    }
}

File: Tareas_FdP/Tarea02vscode/Ejercicio01.py
# Ejercicio 1: Ganancia por Inversión

print("Ingrese el monto a invertir: ")
capital = float(input())

print("Ingrese el número total de días del mes a considerar: ")
dias = int(input())

interes = 0.02
ganancia = capital * dias * interes

print("La ganancia por cobrar después del mes es de:", ganancia)